template/pyproject.toml.jinja

```diff
@@ -93,6 +93,8 @@ fix = true
 
 [tool.ruff.lint]
 ignore = [
+    "D203", # one-blank-line-before-class and `no-blank-line-before-class` (D211) are incompatible.
```

Comment:
These ignore selections requires update  of some of your docstrings. I don't mind using other rule. But we should align with Param. See https://github.com/holoviz/param/pull/982.